bahmni-commons-api/src/main/java/org/bahmni/module/bahmnicommons/contract/patient/search/PatientNameQueryHelper.java

```diff
@@ -6,7 +6,7 @@
 
 public class PatientNameQueryHelper {
 	private String name;
-	public static final String BY_NAME_PARTS = " concat_ws(' ',coalesce(given_name), coalesce(middle_name), coalesce(family_name)) like ";
+	public static final String BY_NAME_PARTS = " concat_ws(' ',coalesce(pn.given_name), coalesce(pn.middle_name), coalesce(pn.family_name)) like ";
```

Comment:
this class is also redundant and used only in the deprecated call chain of PatientDaoImpl.getPatient()